app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= pb_rails("phone", props: {
+  icon: "mobile-android",
+  number: "(349) 185-9988"
```

Comment:
Can you add a formatted in the kit class that will take a number, strip out any non-numeric characters, then use `number_to_phone(raw_numbers, area_code: true)` to make the number look good?

Reply:
Add a number formatter to the number method in phone.rb